### ISSUE TITLE
run custom command

### ISSUE THREAD

**Issue by stav242** [hjelev/rpi-mqtt-monitor#181]
Hi,

[feature request] 

could you add ontio the future feature list, the ability to run a custom script/program remotlely, triggered over MQTT.

i would like to restart just the chrome browser if i see it using up too much ram.
and im sure this feature  @could be used by others.

**Comment by pallago** [comment]
I like this idea and would also like to contribute. I would prefer a list with

- custom command(s) (scripts)
- custom icon(s)
- custom MQTT topic(s) / payloads

like `[["/mypath/myscript1.sh", "mdi:myicon1", "mytopic1"], ["/mypath/myscript2.sh", "mdi:myicon2", "mytopic2"]]`

any preferences?

**Comment by LadnerJamie** [comment]
I agree that this would be a welcome feature. I could use it to change the dpms settings on my Rpi to enable and disable the screen.

**Comment by hjelev** [comment]
After couple of years this feature is now present in version 1.4.3
